FAQ.md

```diff
@@ -0,0 +1,52 @@
+# FAQ - Frequently Asked Questions
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+
+**Table of Contents**
+
+- [FAQ - Frequently Asked Questions](#faq---frequently-asked-questions)
+  - [Installation](#installation)
+  - [Trading](#trading)
+  - [Wallet](#wallet)
+  - [Contact us](#contact-us)
+
+<!-- markdown-toc end -->
+
+## Installation
+
+Q: Do I need to use Umbrel to use Itchysats?
+A: Not at all! While [Umbrel](https://getumbrel.com) is a convenient way to use ItchySats (one-click install from the Umbrel App Store), it is possible to try out itchysats with one of the following methods:
```

Comment:
```suggestion
A: Not at all - but we recommend it for non-tech-savvy users! While [Umbrel](https://getumbrel.com) is a convenient way to use ItchySats (one-click install from the Umbrel App Store), it is possible to try out itchysats with one of the following methods:
```

Reply:
I would say that docker/binary is on-par, especially recently. We mention Umbrel is convenient, but at the same I would argue that we don't want to make it unnecessarily hard to try it out (e.g. having a spare RPi at home, when the pandemic wiped the stores clean).